src/network/core/tcp.cpp

```diff
@@ -31,15 +31,19 @@ NetworkTCPSocketHandler::~NetworkTCPSocketHandler()
 {
 	/* Virtual functions get called statically in destructors, so make it explicit to remove any confusion. */
 	this->NetworkTCPSocketHandler::CloseConnection();
+	this->NetworkTCPSocketHandler::CloseSocket();
```

Comment:
Since `CloseSocket` is not virtual anymore, the explicit class is not needed. For `CloseConnection` it is still needed though.